pkg/aws/client.go

```diff
@@ -431,6 +435,47 @@ func (c *awsClient) GetRegion() string {
 	return aws.StringValue(c.awsSession.Config.Region)
 }
 
+func (c *awsClient) IsPublicSubnet(subnet *ec2.Subnet) (bool, error) {
+	routeTablesResp, err := c.ec2Client.DescribeRouteTables(&ec2.DescribeRouteTablesInput{
```

Comment:
I wonder if we should fetch all route tables at once to reduce API calls, then we can have a subnet -> route table map, WDYT?

Reply:
I'm actually not sure if takes longer to retrieve all route tables or filter through subnets :thinking: . Will test a bit before going back to this

Reply:
It could be a premature optimization, if it takes a reasonable time to check one by one, we can continue with the current approach.

Reply:
Oh, I meant retrieve all route tables providing all the provided subnets, not all the route tables in the AWS account.
The subnet input field is of type `[]*string`, we can pass multiple subnets to filter.

Reply:
Feels faster! Thanks for the input